doc/en/user/source/community/geofence-server/tutorial.rst

```diff
@@ -0,0 +1,127 @@
+Using the Internal GeoFence server (Tutorial)
+=============================================
+
+Introduction
+------------
+
+This tutorial shows how to install and configure the :ref:`community_geofence_server` plug-in. It shows how to create rules in two ways: using the GUI and REST methods.
+
+The tutorial assumes:
+
+* GeoServer is running on http://localhost:8080/geoserver
+
+* You have a user/group service called "default" that allows the creation of new users. If your primary user/group service is not called "default", create a new text file called '/geofence/geofence-server.properties' in the geoserver data directory and add the following line::
```

Comment:
Recommend either ``double ticks`` or :file: notation for paths instead of single quotes.